### PR TITLE
179D - Adjust VAV Fan Power coefficients - G3.1.3.15 VAV fan part-load fan power equation

### PR DESCRIPTION
**Pull request by jmarrec**
Open a PR from @brianlball 's changes in https://github.com/NREL/openstudio-bem-to-surrogate-gem/pull/183

Actual changes to review are in this commit: https://github.com/jmarrec/openstudio-standards/pull/25/commits/aa39ac5e5d7c26f516819993451a12c6f67be853

Redo of #24

### CODE REVIEW

**Comment by brianlball** [comment]
shouldnt it have `class ACM179dASHRAE9012007`  and `include Fan` in it?

File: lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/179d_ashrae_90_1_2007/179d_ashrae_90_1_2007.FanVariableVolume.rb [hunk 1]
```diff
@@ -0,0 +1,110 @@
+class Standard
```
Comment (jmarrec) [inline lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/179d_ashrae_90_1_2007/179d_ashrae_90_1_2007.FanVariableVolume.rb]:
```suggestion
class ACM179dASHRAE9012007
```

**Comment by brianlball** [comment]
using original https://github.com/jmarrec/openstudio-standards/pull/24

**Comment by jmarrec** [comment]
Why? This had two separate commits and does not copy 3 or 4 functions that aren't changed.

**Comment by jmarrec** [comment]
I replayed these onto 179D to get a clean history.

**Pull request closed without merging**